### PR TITLE
Testimonial section for homepage

### PR DESCRIPTION
**Pull request by apeltzer**
I added a short testimonial section to the homepage - can be / should be polished but should already fit in well to the existing homepage hopefully 👍

### CODE REVIEW

**Comment by apeltzer** [comment]
Bildschirmfoto 2019-08-05 um 11 11 13

File: public_html/testimonials.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+$title = 'nf-core testimonials';
+$subtitle = 'Read about the different people and projects relying on nf-core';
```
Comment (drpatelh) [inline public_html/testimonials.php]:
```suggestion
$subtitle = 'Read about those who use/contribute to nf-core pipelines';
```

File: public_html/testimonials.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+include('../includes/header.php');
+?>
+
+<p>We are collecting statements, projects and other peoples (public) comments on how nf-core contributed to their success in analyzing their data. If you know about something that we should probably list here, please let us know!</p>
```
Comment (drpatelh) [inline public_html/testimonials.php]:
```suggestion
<p>We are collating statements and general comments from those who have either contributed to nf-core, or have chosen to routinely deploy nf-core pipelines for their data analysis. If you aware of anything we should include here please let us know!</p>
```

File: public_html/testimonials.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<h1>
+<img height="45px" src="/assets/img/dfg_logo.svg" class="float-right bg-white pl-4" />
+  German National Sequencing Initiative
+</h1>
+<p><a href="https://www.dfg.de/en/service/press/press_releases/2018/press_release_no_06/index.html">The German Funding Body (DFG)</a> for establishing four national high-throughput sequencing centers has started to rely on nf-core pipelines for analyzing large-scale genomics data. Contributors from Kiel and Tübingen already actively contributed to the project, the other sequencing centers in Cologne/Bonn (West German Genome Center) and the Dresden Center are in the process of joining and contributing their expertise.</p>
```
Comment (drpatelh) [inline public_html/testimonials.php]:
```suggestion
<p><a href="https://www.dfg.de/en/service/press/press_releases/2018/press_release_no_06/index.html">The German Funding Body (DFG)</a> has approved funding to establish 4 national high-throughput sequencing centers in Germany. The project will rely on nf-core pipelines for analyzing large-scale genomics data. Contributors from the Kiel and Tübingen sites are already actively contributing to nf-core, and the other sequencing centers in Cologne/Bonn (West German Genome Center) and the Dresden Center are in the process of joining and contributing their expertise too!</p>
```

**Comment by apeltzer** [comment]
> Looks good.
> Do we intend to do that manually every time?
> Or is this a start for more?

I think its more a start for something else 👍

**Pull request merged**